src/common/reducers/webhook.js

```diff
@@ -12,12 +12,26 @@ const ERROR_MESSAGES = {
   'github-error-other': 'A problem occurred while the repository was being built. Please try again later'
 };
 
-export function webhook(state = INITIAL_STATE, code) {
-  switch (code.type) {
-    case WEBHOOK_FAILURE:
+export function webhook(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case ActionTypes.WEBHOOK:
       return {
         ...state,
-        error: { message:  ERROR_MESSAGES[code.code] }
+        isPending: true
```

Comment:
I believe it has the same meaning as `isFetching` that we use in other reducers, so I would rename it to `isFetching` too to keep it consistent.